iaso/api/org_unit_change_requests/views.py

```diff
@@ -126,3 +129,49 @@ def partial_update(self, request, *args, **kwargs):
 
         response_serializer = OrgUnitChangeRequestRetrieveSerializer(change_request)
         return Response(response_serializer.data)
+
+    @staticmethod
+    def org_unit_change_request_csv_columns():
+        return [
+            "Id",
+            "Name",
+            "Parent",
+            "Org unit type",
+            "Groups",
+            "Status",
+            "Created",
+            "Created by",
+            "Updated",
+            "Updated by",
+        ]
+
+    @action(detail=False, methods=["get"])
+    def export_to_csv(self, request):
+        filename = "%s--%s" % ("review-change-proposals", datetime.now().strftime("%Y-%m-%d"))
+        org_unit_changes_requests = self.get_queryset().order_by("org_unit__name")
+        filtered_org_unit_changes_requests = OrgUnitChangeRequestListFilter(
+            request.GET, queryset=org_unit_changes_requests
+        ).qs
+
+        response = HttpResponse(content_type=CONTENT_TYPE_CSV)
+
+        writer = csv.writer(response)
+        headers = self.org_unit_change_request_csv_columns()
+        writer.writerow(headers)
+        for change_request in filtered_org_unit_changes_requests:
+            row = [
+                change_request.id,
+                change_request.org_unit.name,
+                change_request.org_unit.parent.name if change_request.org_unit.parent else None,
+                change_request.org_unit.org_unit_type.name,
+                ",".join(group.name for group in change_request.org_unit.groups.all()),
+                change_request.status,
```

Comment:
```suggestion
                change_request.get_status_display(),
```

There is a "hidden" method to access a formatted string.